lib/primer/classify.rb

```diff
@@ -152,98 +159,102 @@ def validated_class_names(classes)
       # Example usage:
       # extract_hash({ mt: 4, py: 2 }) => "mt-4 py-2"
       def extract_hash(styles_hash)
-        out = styles_hash.each_with_object({ classes: [], styles: [] }) do |(key, value), memo|
+        memo = { classes: [], styles: String.new }
```

Comment:
any reason to use `String.new` instead of `""`?

Reply:
`String.new` lets me mutate the string which we do below, but `""` is frozen due to `# frozen_string_literal: true`